ArcGISRuntimeSDKQt_QMLSamples/Geometry/ClipGeometry/ClipGeometry.qml

```diff
@@ -0,0 +1,198 @@
+// [WriteFile Name=ClipGeometry, Category=Geometry]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
+import Esri.ArcGISRuntime 100.3
+import Esri.ArcGISExtras 1.1
+
+Rectangle {
+  id: rootRectangle
+  clip: true
+  width: 800
+  height: 600
+
+  property real scaleFactor: System.displayScaleFactor
+
+  MapView {
+    id: mapView
```

Comment:
for code spacing, we do 4 spaces for QML and 2 for C++. You can update your settings in Qt Creator Preferences for Code Style under C++ and Qt Quick.

Reply:
This should be corrected now.